src/clr-angular/forms/checkbox/checkbox-wrapper.ts

```diff
@@ -11,12 +11,11 @@ import { DynamicWrapper } from '../../utils/host-wrapping/dynamic-wrapper';
 import { ControlIdService } from '../common/providers/control-id.service';
 import { ClrLabel } from '../common/label';
 
-/* tslint:disable-next-line:variable-name */
-export const TogglerService = new InjectionToken<any>(undefined);
-/* tslint:disable-next-line:variable-name */
-export function TogglerServiceProvider() {
+export const IS_TOGGLE = new InjectionToken<boolean>('IS_TOGGLE');
```

Comment:
Careful, this is `InjectionToken<Subject<boolean>>` or `InjectionToken<BehaviorSubject<boolean>>`, depending on how specific you want to be.

I would expect AoT compilation to fail on this, not sure if it does.

Reply:
Updated